App/target/classes/META-INF/resources/form.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
```

Comment:
Why is this in classes/META-INF/resources instead of scr/main/webapp? What's the difference between this and the other HTML files?

Reply:
Ahhh this is /target, I assume they're just automatically copied here or something..?

Reply:
Yeah, I think it's automatically generated (or something similar), either way, git ignored 👍